### ISSUE TITLE
The import com.codepath.simplegame cannot be resolved

### ISSUE THREAD

**Issue by srichakradhar** [codepath/android_snake_game#3]
There are no source files for

``` java
import com.codepath.simplegame.AbstractGamePanel;
import com.codepath.simplegame.actors.PositionedActor;
```

**Comment by zeroxia** [comment]
Where is the android_simple_game_engine???

**Comment by nesquena** [comment]
It was never published since it's not quite ready. Will add you so you have access.

**Comment by twokwy** [comment]
Also went looking for android_simple_game_engine only to be disappointed :(

**Comment by javand** [comment]
@nesquena Could you add me as well please?

**Comment by nesquena** [comment]
Yep added. Keep in mind this is mostly just a prototype and is not really ready for primetime which is why the repo is private but we appreciate any contributions to improve it.

**Comment by sungkwoneom** [comment]
Hello,

I'm trying to build "Simple 2D Snake Game for Android."
BUT I could not find com.codepath.simplegame.\* library.

Even I try to access "https://github.com/codepath/android_simple_game_engine"  

It just show 404 not found.

How can I get com.codepath.simplegame.\* library  ?

thk in advance.

**Comment by kerwinwalker** [comment]
android_simple_game_engine not found. 😥

**Comment by shrekmaster** [comment]
Please add me to access simplegame library

**Comment by nesquena** [comment]
You've both been invited

**Comment by Thelouras58** [comment]
Hey, can you send me a link for the to android_simple_game_engine?
Thanks!

**Comment by aman4india** [comment]
where is the library?

**Comment by monashri** [comment]
Can you please share the library "**android_simple_game_engine**" ? . I wish to try and develop this game .

**Comment by nesquena** [comment]
I've now made the repo public, I am surprised by how many people find this repo and request access: https://github.com/codepath/android_simple_game_engine. Keep in mind this repository is not currently maintained and is a simple proof of concept. Feel free to improve this code but keep in mind that this is likely not to work for anything beyond the most of basic use cases.
